core-bundle/src/Controller/SitemapController.php

```diff
@@ -95,6 +95,9 @@ public function __invoke(Request $request): Response
         $response = new Response((string) $sitemap->saveXML(), 200, ['Content-Type' => 'application/xml; charset=UTF-8']);
         $response->setSharedMaxAge(2592000); // will be unset by the MakeResponsePrivateListener if a user is logged in
 
+        // Make sure an authorized request does not retrieve the sitemap from the HTTP cache (see #6832)
```

Comment:
```suggestion
        // Make sure an authorized request does not retrieve the sitemap from the HTTP cache (see #6832)
        // Do not upstream merge this into Contao 5!
```

This makes it more likely that it will not be overlooked. 🙈

Reply:
I'll do the PR for Contao 5 first and leave this conversion open until then. This might need to be discussed first, before we apply those changes in Contao 5.

Reply:
As discussed today we do not want to change the implementation in Contao 5, thus this can be merged upstream.